### PR TITLE
Add note about timeout parameters

### PR DESCRIPTION
**Pull request by nataliiakovalova**

### CODE REVIEW

**Comment by DarkEld3r** [comment]
I propose to add a note that generally it makes sense to make `round_timeout` twice bigger than `consensus.timeout_adjuster.timeout`.

**Comment by nataliiakovalova** [comment]
> I propose to add a note that generally it makes sense to make round_timeout twice bigger than consensus.timeout_adjuster.timeout.
@DarkEld3r  done

File: src/advanced/configuration-updater.md [hunk 1]
```diff
@@ -122,6 +122,12 @@ config][stored_configuration] serialization. It has the following fields:
   Maximum number of transactions per block.
 - **services**: Object  
   Service-specific configuration parameters.
+  
+!!! note
+    **consensus.round_timeout** integer must be strictly larger 
+    (recommended at least 2x) than **consensus.timeout_adjuster.timeout** 
+    integer, otherwise, the consensus algorithm will stop working correctly
```
Comment (slowli) [inline src/advanced/configuration-updater.md]: I'd suggest to split the "otherwise..." part into a new sentence to improve readability.

Comment (nataliiakovalova) [reply src/advanced/configuration-updater.md]: @slowli Thanks. Done.

**Pull request merged**